libs/ui/src/lib/symbol-autocomplete/symbol-autocomplete.component.ts

```diff
@@ -132,7 +133,9 @@ export class SymbolAutocompleteComponent
   public ngDoCheck() {
     if (this.ngControl) {
       this.validateRequired();
-      this.validateSelection();
+      if (this.control.touched) {
```

Comment:
Is this the whole fix for setting the value from outside?

Reply:
Yes this is enough. The issue was since when we are setting the value from outside the input is not touched so no API call is made to fetch the list of items. So to avoid that I have skipped the check until anyone makes changes to the input